### ISSUE TITLE
USWDS - Feature: Enhance Validation screen reader initial callouts

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds#5840]
### Is your feature request related to a problem? Please describe.

Follow up to #5760 

If an AT user navigates via tabbing and finds themself on the validation component, it does not read the current state of the validation requirements. 

Originally, we added `tabindex` to ensure the user tabs through the requirements before arriving to the input that is being validated. In #5760 we decided against this pattern as it goes against convention to allow no interactive elements to be tabbed to.

### Describe the solution you'd like

The screen reader should read out the validation requirements when users first arrive to the input, like it does after input change.

### Additional context

During initialization, we create a sr-only span which will receive the validation requirements after users input text into the text input. It would be ideal if this text was added upon initialization rather after input.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature+Request%22) for duplicate feature requests.

**Comment by mahoneycm** [comment]
Closing in favor of #5841
